pennylane_sf/gbs.py

```diff
@@ -24,11 +24,10 @@
 import strawberryfields as sf
 from strawberryfields.ops import GraphEmbed, MeasureFock
 from strawberryfields.utils import all_fock_probs_pnr
-from thewalrus.quantum import find_scaling_adjacency_matrix as rescale
+from thewalrus.quantum import adj_scaling as rescale
```

Comment:
Is this due to an update to `thewalrus`?

Reply:
yep, I was getting a deprecation notice 😆